### PR TITLE
feat: added input component

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (533)
Screenshot (534)
Screenshot (535)
Screenshot (536)

### CODE REVIEW

**Review by shivampandey0 — COMMENTED** [review]
short and easy-to-understand code.
Good to merge.

File: Components/input.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    max-width: 1000px;
+}
+
+.input-box {
+    padding: .5em 0;
+    margin: 10px;
+    width: 30VW;
```
Comment (Hamdrive) [inline Components/input.css]: use px and rems where necessary

Comment (Hamdrive) [inline Components/input.css]: also units should be in smallcase

**Review by Hamdrive — COMMENTED** [review]
Good work done, ,changes suggested to be made

File: Docsite-HTML-Files/input.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                style="width: 700px; height: 282px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <!-- range -->
+            <h3 class="spacer-2rem">Example 4: <strong>Slider with type = "range"</strong> </h3>
+
+            Rohan Dubey, [15-02-2022 11:21 PM]
+            [Forwarded from Rohan Dubey]
```
Comment (MaruthiPothuganti) [inline Docsite-HTML-Files/input.html]: Typo

**Review by MaruthiPothuganti — COMMENTED** [review]
Apart from what shivam and Hamza mentioned, there is a typo. Everything else looks good

**Pull request merged**